app/controllers/registrar/xml_consoles_controller.rb

```diff
@@ -18,8 +19,82 @@ def load_xml
       cl_trid = "#{depp_current_user.tag}-#{Time.zone.now.to_i}"
       xml_dir_path = Rails.root + 'app/views/registrar/xml_consoles/epp_requests'
       xml = File.read("#{xml_dir_path}/#{params[:obj]}/#{params[:epp_action]}.xml")
-      xml.gsub!('<clTRID>ABC-12345</clTRID>', "<clTRID>#{cl_trid}</clTRID>")
+      xml = insert_schema_path(xml, cl_trid)
```

Comment:
We are not only inserting schema path here, but cl_trid too, so I'd suggest to rename method to something like `prepare_payload`